docs/getting-started-guides/juju.md

```diff
@@ -65,17 +65,17 @@ docker run --rm -ti \
 
 ### Configure Juju to your favorite cloud provider
 
-At this point you have access to the Juju client. Before you can deploy a
+At this point, you have access to the Juju client. Before you can deploy a
```

Comment:
"Once you have mounted the container, you should have access to the Juju client."